doc/input.md

```diff
@@ -95,20 +95,42 @@ and [MouseRegion widget](https://api.flutter.dev/flutter/widgets/MouseRegion-cla
 also read more about Flutter's gestures
 [here](https://api.flutter.dev/flutter/gestures/gestures-library.html).
 
+## Event coordinate system
+
+On events that have positions, like for example `Tap*` or `Drag`, you will notice that the `position`
+attribute includes 3 fields: `game`, `widget` and `global`. Below you will find a brief explanation
+about each one of them.
+
+### global
+
+Coordinates with the position where the event occurred considering the entire screen, same as
```

Comment:
I think you can remove the first part and it will be easier to read?
Just skip "Coordinates with".

Reply:
On the other sections too

Reply:
done